.github/scripts/build_instance/main.go

```diff
@@ -65,7 +76,38 @@ func createRun(ctx context.Context, client *tfe.Client) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to trigger run: %w", err)
 	}
+	log.Printf("Run created: %s", runID)
 
-	log.Printf("Created run: %s\n", run.ID)
 	return run.ID, nil
 }
+
+func waitForRun(ctx context.Context, client *tfe.Client, runID string) error {
+	// The run should take about 5 minutes to complete;
+	// polling the status of the run every 20 seconds or so
+	// should be frequent enough. It's also long enough to ensure
+	// no ticks are dropped.
+	ticker := time.NewTicker(time.Second * 20)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return fmt.Sprintf("Context canceled: %w", ctx.Err())
```

Comment:
Is this supposed to be...

```suggestion
			return fmt.Errorf("Context canceled: %w", ctx.Err())
```
?

And if so, how did this compile?

Reply:
Sloppy on my part :disappointed: , I've relied on the static analyzer and it seems to "break" due to `goenv` (uses `$GOPATH/{version_number}/bin` and I forget to reload my shell so it can use the correct go version).